benchmark/benchmark_measurements.py

```diff
@@ -0,0 +1,34 @@
+from pyrsistent import PClass, field
```

Comment:
Since there's no actual thing in Python (e.g. a class) that defines what a measurement is, the docstring for this module should define it (or maybe there should be an interface). 

AFAICT, a measurement is a callable that takes a function and parameters, runs the function, discards the return value and returns the, umm, measurement, which has to be JSON-serializable. If the function raises an error, then that bubbles up. 

Is that right?